src/engines/aurora/gui.cpp

```diff
@@ -217,6 +219,8 @@ void GUI::callbackRun() {
 void GUI::callbackActive(Widget &UNUSED(widget)) {
 }
 
+void GUI::callbackTextInput(Common::UString text) {
```

Comment:
Please wrap the `text` in the `UNUSED()` macro here, since the parameter isn't used in this method.